### PR TITLE
Add function to fetch raw openscap arf

### PR DESCRIPTION
**Pull request by enoodle**
Based on: simon3z/image-inspector#3

### CODE REVIEW

**Comment by enoodle** [comment]
@moolitayer

File: test/test_openscap.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  def test_openscap_v1
+    stub_request(:get, %r{/openscap}).to_return(
+      body: open_test_file('openscap.xml'),
+      status: 200
+    )
+
+    md = ImageInspectorClient::Client.new('http://localhost:8080', 'v1')
+         .fetch_openscap
+
+    assert_instance_of(String, md)
```
Comment (moolitayer) [inline test/test_openscap.rb]: test that md is the expected string?

**Comment by moolitayer** [comment]
depends on https://github.com/simon3z/image-inspector/pull/3
@enoodle look fine. 
Could you add a section in the README (Usage part)? it should be clear that raw file content is being returned

File: README.md [hunk 2]
```diff
@@ -11,6 +11,12 @@ ImageInspectorClient::Client.new('http://localhost:8080', 'v1')
   .fetch_metadata
   .ContainerConfig
   .Cmd
+
+
+To get the openscap report as raw xml:
```
Comment (moolitayer) [inline README.md]: Could you please change above to

# Get OpenSCAP ARF as raw xml

File: lib/image-inspector-client.rb [hunk 3]
```diff
@@ -28,6 +28,12 @@ def initialize(
       @headers = {}
     end
 
+    def fetch_openscap
+      exception_handler do
+        RestClient::Resource.new(@endpoint, http_options)['openscap'].get(http_headers).body
```
Comment (simon3z) [inline lib/image-inspector-client.rb:33]: @moolitayer @enoodle you're not doing the XML parsing here?

File: README.md [hunk 4]
```diff
@@ -11,6 +11,12 @@ ImageInspectorClient::Client.new('http://localhost:8080', 'v1')
   .fetch_metadata
   .ContainerConfig
   .Cmd
+
+
+Get OpenSCAP ARF as raw xml:
+ImageInspectorClient::Client.new('http://localhost:8080', 'v1')
+  .fetch_openscap
```
Comment (moolitayer) [inline README.md]: fetch_oscap_arf

Comment (enoodle) [inline README.md]: Do you want to change the function name to `fetch_oscap_arf`?

**Comment by enoodle** [comment]
@moolitayer I changed the error handler to record the failure explanation.

**Comment by moolitayer** [comment]
@enoodle two nit comments for test_openscap.rb:
- you can remove all the assert_instance_of now that you test the strings
- at test_openscap_404 please test the content of e.message

Thanks!

**Comment by moolitayer** [comment]
Thank you @enoodle :tada: :star2:

**Pull request merged**